requirements.txt

```diff
@@ -1,13 +1,13 @@
 # This file is only used for running Github actions. Requirements that are installed with a project are listed in
 # setup.py install_requires.
 
-pandas==1.2.5
-anndata==0.8.0
-scanpy==1.9.1
-numpy==1.21
-requests==2.22.0
-boto3==1.21.27
+pandas>=1.2.5
+anndata>=0.8.0
+scanpy>=1.9.1
+numpy>=1.21
+requests>=2.22.0
+boto3>=1.21.27
```

Comment:
and we didn't remove pandas and anndata and scanpy and numpy from here, so it is not consistent with the toml